summary_workflows/identification/nextflow.config

```diff
@@ -66,10 +70,10 @@ params  {
   participant_id = "DEF_PARTICIPANT"
 
   // general data directory
-  data_dir = "$baseDir/APAeval_I_test_data"
+  data_dir = "$launchDir/../DATA"
   
   // directory where the 'gold standards' are found
-  goldstandard_dir = "${params.data_dir}/ref"
+  goldstandard_dir = "${params.data_dir}/ground_truths"
```

Comment:
I wouldn't want to merge this, as it breaks compatibility with the APAeval test file paths